sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/util/DateTimeTestUtils.scala

```diff
@@ -22,20 +22,31 @@ import java.util.TimeZone
 import java.util.concurrent.TimeUnit
 
 import org.apache.spark.sql.catalyst.util.DateTimeConstants._
+import org.apache.spark.sql.catalyst.util.DateTimeUtils.getZoneId
 
 /**
  * Helper functions for testing date and time functionality.
  */
 object DateTimeTestUtils {
 
+  val CEST = getZoneId("+02:00")
+  val CET = getZoneId("+01:00")
+  val JST = getZoneId("+09:00")
```

Comment:
if 3 letter short name is ambiguous, shall we pick a clear name like `JAPAN_ZONE`?

Reply:
I haven't found alternative definition of `JST` in https://www.timeanddate.com/time/zones/

The problem of `JST` is it maps to `Asia/Tokyo`, see
https://github.com/frohoff/jdk8u-jdk/blob/master/src/share/classes/java/time/ZoneId.java#L241
And `Asia/Tokyo` had daylight saving time transitions before 8-Sep-1951.

Since `JST` is `+09:00` by definition, the changed line reflects this.